staging/src/slime.io/slime/modules/meshregistry/pkg/bootstrap/args.go

```diff
@@ -171,6 +171,9 @@ type NacosSourceArgs struct {
 	Password string
 	// fetch services from all namespaces, only support Polling mode
 	AllNamespaces bool
+	// If enabled, the namespace and group information will be injected into ep's metadata.
+	// Use env NACOS_NAMESPACE_META_KEY and NACOS_GROUP_META_KEY to configure the keys to be injected.
+	InjectNsGroupIntoMeta bool
```

Comment:
The combined use of env and module configuration is too heavy-handed for this simple scenario.

`MetaKeyGroup, MetaKeyNamespace string`

```go
if xx.MetaKeyGroup != "" {
  inst.Metadata[xx.MetaKeyGroup] = namespaceId
}
```

Reply:
replaced `MetaKeyGroup, MetaKeyNamespace string`